include/fiction/io/write_operational_domain.hpp

```diff
@@ -103,14 +103,17 @@ namespace detail
  * The operational status is a binary value represented by specified tags in `params` indicating whether the simulation
  * parameters are within the operational domain or not.
  *
+ * @tparam OpDomain The type of the operational domain.
  * @param opdom The operational domain to be written. It contains a mapping from sets of simulation parameters
- * (represented as a pair of sweep parameters for the X and Y dimensions) to their operational status.
+ * (represented as a pair of sweep parameters for the X and Y dimensions) to their operational status, and it can
+ include metric values (e.g., the critical temperature on top of that).
```

Comment:
What does it mean that it can "include" metric values? How are those represented? Can we get an example file excerpt here?